packages/javascript/bh-shared-ui/src/components/CreateUserForm/CreateUserForm.tsx

```diff
@@ -14,426 +14,696 @@
+            ...(selectedETACEnabledRole === false ? formData : eTACFormData),
+        });
     };
 
+    useEffect(() => {
+        // on form load error
+        if (error) {
+            form.setError('root.generic', {
+                type: 'custom',
+                message: 'An unexpected error occurred. Please try again.',
+            });
+        }
+    }, [form, error, form.formState.errors]);
+
     return (
-        <form autoComplete='off' onSubmit={handleSubmit(onSubmit)}>
-            {!(getListDisplayRolesQuery.isLoading || listSSOProvidersQuery.isLoading) && (
-                <>
-                    <DialogContent>
-                        <Grid container spacing={2}>
-                            <Grid item xs={12}>
-                                <Controller
-                                    name='emailAddress'
-                                    control={control}
-                                    rules={{
-                                        required: 'Email Address is required',
-                                        maxLength: {
-                                            value: MAX_EMAIL_LENGTH,
-                                            message: `Email address must be less than ${MAX_EMAIL_LENGTH} characters`,
-                                        },
-                                        pattern: {
-                                            value: /^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$/,
-                                            message: 'Please follow the example@domain.com format',
-                                        },
-                                    }}
-                                    render={({ field }) => (
-                                        <TextField
-                                            {...field}
-                                            variant='standard'
-                                            id='emailAddress'
-                                            label='Email Address'
-                                            type='email'
-                                            fullWidth
-                                            error={!!errors.emailAddress}
-                                            helperText={errors.emailAddress?.message}
-                                            data-testid='create-user-dialog_input-email-address'
-                                        />
-                                    )}
-                                />
-                            </Grid>
+        <Form {...form}>
+            <form
+                autoComplete='off'
+                data-testid='create-user-dialog_form'
+                onSubmit={form.handleSubmit(handleOnSave, onError)}>
+                {!(getRolesQuery.isLoading || listSSOProvidersQuery.isLoading) && (
+                    <div className='flex gap-x-4 justify-center'>
+                        <Card className='p-6 rounded shadow max-w-[600px] w-full'>
```

Comment:
Same comment regarding Card styling inside the Dialog: https://github.com/SpecterOps/BloodHound/pull/1858/files#r2474218509